Automattic/wp-calypso
# WooCommerce Install Landing Page Cleanup - Remove unused images

#### Changes proposed in this Pull Request

* Removes unused images

#### Testing instructions



* Visit `http://calypso.localhost:3000/woocommerce-installation/<site-id>`.
* Make sure the flow works.



Related to #59190
